### ISSUE TITLE
FR: Move screen with one finger on ipad

### ISSUE THREAD

**Issue by AsleepX** [zsviczian/obsidian-excalidraw-plugin#1672]
I've tried using Excalidraw on the iPad🫡, but having to press a button to enable the pan function feels a bit awkward to me😔. I wish I could move the canvas around with one finger🥰, just like in other handwriting note-taking apps on the iPad🤯.

**Comment by zsviczian** [comment]
Yes, fully agree. This request has been out there for a long time.  However, this should be addressed in the core Excalidraw package, not the Obsidian plugin. 

Perhaps you can open a PR to solve this with excalidraw? https://github.com/excalidraw/excalidraw

**Comment by AsleepX** [comment]
I'll try to bring this up in excaldraw, and thank you for your continued efforts with this plugin!
